modules/RTC/TraceablePeerConnection.js

```diff
@@ -1875,7 +1894,7 @@ TraceablePeerConnection.prototype.setLocalDescription = function(description) {
                 }
                 resolve();
             }, err => {
-                this.trace('setLocalDescriptionOnFailure', err);
+                this.trace('setLocalDescription::OnFailure', err);
```

Comment:
Interesting, why the double dots :: ?

Reply:
for filter in console.. the other use it as well so it is consistence